tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -105,6 +98,14 @@
 
 data_cov.plot(epochs.info)
 
+# When looking at the covariance matrix plots, we can see that our data is
```

Comment:
you'll want a row of `###` above this, so it gets formatted as text instead of code comments.